### ISSUE TITLE
No arg for -v option (install.sh missing version number)

### ISSUE THREAD

**Issue by martinb3** [test-kitchen/test-kitchen#900]
Hello! I was just testing the current master (appears to be a 1.5 RC), and I'm getting an error on the chef zero provisioner. Specifically, it's passing an empty version string to install.sh:

```
-----> Installing Chef Omnibus ()
       Downloading https://www.chef.io/chef/install.sh to file /tmp/install.sh
       Trying wget...
       Download complete.
       No arg for -v option
       usage: /tmp/install.sh [-P project] [-c release_channel] [-v version] [-f filename | -d download_dir]
>>>>>> Converge failed on instance <package-ubuntu-1404>.
>>>>>> Please see .kitchen/logs/package-ubuntu-1404.log for more details
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: SSH exited (1) for command: [sh -c '

chef_omnibus_root="/opt/chef"
chef_omnibus_url="https://www.chef.io/chef/install.sh"
install_flags="-v"
pretty_version=""
sudo_sh="sudo -E sh"
version=""
```

I was able to bisect it back to [this change](https://github.com/test-kitchen/test-kitchen/commit/7318fa5bb93319b68303b0f93302f1c35bb8e562) in ` lib/kitchen/provisioner/chef_base.rb`:
```
+        # Passing "true" to mixlib-install currently breaks the windows metadata_url
+        # TODO: remove this line once https://github.com/chef/mixlib-install/pull/22
+        # is accepted and released
+        version = "" if version == "true"
```

I'm not sure why this is broken (I'll debug further now), but I have a very simple `.kitchen.yml` for reproducing it:
```
---
driver:
  name: vagrant # provide a default test-kitchen driver, vagrant

provisioner:
  name: chef_zero
  nodes_path: 'test/fixtures/nodes'
  data_bags_path: 'test/fixtures/data_bags'
  environments_path: 'test/fixtures/environments'
  client_rb:
    environment: _default
  attributes:
    java:
      jdk_version: '7'

platforms:
  - name: ubuntu-14.04
    run_list:
    - recipe[apt]
    - recipe[curl]

suites:
  - name: package # install from package
    require_chef_omnibus: 12.6.0
    run_list:
      - recipe[java]
      - recipe[elasticsearch_test::default_with_plugins]
    attributes:
      elasticsearch:
        install_type: package
```

**Comment by martinb3** [comment]
/CC @tyler-ball @mwrock since they were involved in the original commit that introduced this line.

**Comment by mwrock** [comment]
This should be fixed by #899 once its pulled in

**Comment by cheeseplus** [comment]
Closed by #899
